flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitor.java

```diff
@@ -139,7 +148,14 @@ else if (flinkRoot != null) {
 			throw new IllegalConfigurationException("The path to the static contents (" + 
 					webRootDir.getAbsolutePath() + ") is not a readable directory.");
 		}
-		
+
+		final File logStdoutDir = new File(flinkRoot, LOG_OUTPUT_DIRECTORY);
```

Comment:
please use the same kind of checks as for the resources. its fails for my machine after building:
```
org.apache.flink.configuration.IllegalConfigurationException: The path to the log and stdout directory (/Users/ufuk/code/flink-2/flink-dist/target/flink-0.10-SNAPSHOT-bin/flink-0.10-SNAPSHOT/conf/../log) is not a readable directory.
	at org.apache.flink.runtime.webmonitor.WebRuntimeMonitor.<init>(WebRuntimeMonitor.java:156)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.apache.flink.runtime.jobmanager.JobManager$.startWebRuntimeMonitor(JobManager.scala:1772)
	at org.apache.flink.runtime.jobmanager.JobManager$.runJobManager(JobManager.scala:1322)
	at org.apache.flink.runtime.jobmanager.JobManager$.main(JobManager.scala:1194)
	at org.apache.flink.runtime.jobmanager.JobManager.main(JobManager.scala)
```